research/src/pages/checkbox.proposal.mdx

```diff
@@ -54,34 +54,29 @@ The `<checkbox>` control is primarily leveraged to indicate a binary state. For
 #### DOM Structure
 
 ```html
-<host>
```

Comment:
@gregwhitworth in continue to previous comment. I think better to leave `host` since this is was implicitly hint on Shadow DOM/HTML template concepts. And my understanding that skeletons/anatomy is something similar in the spirit. 

Maybe it would be good to expand what is "host" means somewhere else on the website.

Reply:
But the `oui-checkbox` is actually the host. Maybe, to clarify the confusion here is to have the checkbox-anatomy.js have a class of `host` rather than a tag of `<host>` so that the generated content applies?